src/sentry/deletions/tasks/delete_seer_grouping_records.py

```diff
@@ -18,7 +18,7 @@
 
 
 @instrumented_task(
-    name="sentry.tasks.delete_seer_grouping_records_by_hash",
+    name="sentry.deletions.tasks.delete_seer_grouping_records_by_hash",
```

Comment:
I would avoid changing task names, or at least use `alias` to smooth out deploys.